wordcount.c

```diff
@@ -0,0 +1,15 @@
+#include <stdio.h>
```

Comment:
your code didn't output a wordcount! It got stuck and I had to control+C - i think it has to do with the while loop!